ale_linters/bitbake/oelint_adv.vim

```diff
@@ -37,6 +37,10 @@ function! StripAnsiCodes(line) abort
     return substitute(a:line, '\e\[[0-9;]\+[mK]', '', 'g')
 endfunction
 
+function! RemoveBranch(line) abort
```

Comment:
This function can pollute the global namespace and might conflict with a function a use has defined. I'll merge this and use a function name specific to this file.